src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/TrustCorporateDto.java

```diff
@@ -15,57 +17,81 @@ public class TrustCorporateDto {
     @JsonProperty("date_became_interested_person")
     private LocalDate dateBecameInterestedPerson;
 
+    @JsonProperty("registered_office_address")
+    private AddressDto registeredOfficeAddress;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("ro_address_line_1")
     private String roAddressLine1;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("ro_address_line_2")
     private String roAddressLine2;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("ro_address_care_of")
     private String roAddressCareOf;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("ro_address_country")
     private String roAddressCountry;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("ro_address_locality")
     private String roAddressLocality;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("ro_address_po_box")
     private String roAddressPoBox;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("ro_address_postal_code")
     private String roAddressPostalCode;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("ro_address_premises")
     private String roAddressPremises;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("ro_address_region")
     private String roAddressRegion;
 
+    @JsonProperty("service_address")
```

Comment:
Wondering why we add a 'service address' object as well as all the individual address fields?